pkg/route2httpproxy/route2httpproxy_test.go

```diff
@@ -0,0 +1,111 @@
+package route2httpproxy
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	route "github.com/openshift/api/route/v1"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	core "k8s.io/api/core/v1"
+)
+
+func TestDeployment(t *testing.T) {
+	type output struct {
+		kind       string
+		apiVersion string
+		fqdn       string
+	}
+
+	tests := []struct {
+		name         string
+		routeInput   string
+		serviceInput string
+		want         output
+		domain       string
+	}{
+		{
+			"Test config map with Rolling Strategy",
```

Comment:
The name of this test refers to deploymentconfig data - it should be "route with tls"